resources/assets/js/app/controllers/UserController.js

```diff
@@ -13,5 +15,15 @@ module.exports = Marionette.Object.extend({
         app.render(new UsersView({
             collection: users
         }));
+    },
+
+    showUserVotes: function(id) {
+        var usersVotes = new usersVotesCollection([], {parentUrl: '/users/' + id});
```

Comment:
don't use userId from url, this page should show Vote collection only from currently logged in user, see example mytopics